nomenclature/definitions/variable/economy/economy.yaml

```diff
@@ -184,7 +184,7 @@ Price|Final Energy|Residential|Electricity:
                 Prices should include the effect of carbon prices
                 Mean price should reflect the variability of different prices that are accessible
                 to end-users (including regulated prices, prices proposed by different competiting retailers...)
-   unit: US$2010/GJ
+   unit: [euro/kWh, US$2010/GJ]
```

Comment:
euro without a capital letter?

Reply:
good point, also raised by @sandrinecharousset at #85 - but this should simply be harmonized in a next iteration, this does not need to hold back merging this PR

Reply:
I agree and will be aware of this unit issue for the next PR iteration.